.changeset/moody-crabs-begin.md

```diff
@@ -0,0 +1,8 @@
+---
+"@paprika/input": patch
+"@paprika/radio": patch
+"@paprika/select": patch
+"@paprika/textarea": patch
+---
+
+Refactor to component story format
```

Comment:
If there are no changes to the files in `/src` or `package.json` then the package we release will have no changes, and we don't need to include a `changeset`.